tests/semantic/.env.filip

```diff
@@ -0,0 +1,3 @@
+CB_URL="http://localhost:1026"
```

Comment:
please move tests to tests/semantics

Reply:
File should not exist anymore; now deleted